frontend/pages/admin/pacta-version/new.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { type PactaVersion } from '@/openapi/generated/pacta'
+
+const router = useRouter()
+const { pactaClient } = useAPI()
+const { error: { withLoadingAndErrorHandling, handleOAPIError } } = useModal()
+
+const prefix = 'admin/pacta-version/new'
+const pactaVersion = useState<PactaVersion>(`${prefix}.pactaVersion`, () => ({
+  id: '',
+  name: '',
+  description: '',
+  digest: '',
+  createdAt: '',
+  isDefault: false
+}))
+
+const discard = () => router.push('/admin/pacta-version')
+const save = () => withLoadingAndErrorHandling(
+  () => pactaClient.createPactaVersion(pactaVersion.value).then(handleOAPIError),
+  `${prefix}.save`
+)
+</script>
+
+<template>
+  <StandardContent>
+    <TitleBar title="New PACTA Version" />
```

Comment:
Same note here as in `StandardContent`, every page shouldn't have to specify these components, we can hoist them into the default layout and set the title with a `useTitle` composable or similar.

Reply:
I disagree on this one - mostly because <TitleBar> is likely to contain other stuff (like a 3-dot menu) that benefits from being set up as a normal component (i.e. access to computed values in it's props)